Integrations/integration-cyberark_AIM.yml

```diff
@@ -179,7 +179,7 @@ script:
     };
 
     var getAccountDetails = function(args) {
-        var result = sendRequestWithToken('GET', credentialsUrl, args, true);
+        var result = sendRequestWithToken('GET', credentialsUrl, args);
```

Comment:
Y did U remove it again? I use the error message to parse what account are there in the folder.

Reply:
Thats `sendRequestWithToken` last arg is used to get raw data or parsed. since `reset-credentials` does not return data.

Reply:
ok.